padle/padle.py

```diff
@@ -0,0 +1,485 @@
+        await self.config.user(ctx.author).editID.set(message.id)
+        await self.config.user(ctx.author).channelID.set(ctx.channel.id)
+        score = ":yellow_square:"
+        if(guessMonster.monster_id == monster.monster_id):
+            await ctx.send("You got the PADle in {} guesses! Use `^padle score` to share your score.".format(
+                len(await self.config.user(ctx.author).guesses())))
+            await self.config.user(ctx.author).done.set(True)
+            async with self.config.allScores() as allScores:
+                allScores.append(str(len(await self.config.user(ctx.author).guesses())))
+            score = ":green_square:"
+        else:
+            if(points < 9):
+                score = ":orange_square:"
+            if(points < 5):
+                score = ":red_square:"
+        async with self.config.user(ctx.author).score() as scores:
+            scores.append(score)
+        start_adding_reactions(message, emojis)
+        await menu(ctx, embed_pages, embed_controls, message=message, page=len(embed_pages) - 1, timeout=60 * 60)
+
+    @padle.command()
+    async def giveup(self, ctx):
+        """Give up on today's PADle"""
+        if ctx.guild is not None:
+            await ctx.send("You can only play PADle in DMs!")
+            return
+        if(not await self.config.user(ctx.author).start()):
+            await ctx.send("You have not started the game of PADle yet, try `^padle start`!")
+            return
+        if(await self.config.user(ctx.author).done()):
+            await ctx.send("You have already played today's PADle!")
+            return
+        confirmation = await get_user_confirmation(ctx, "Are you sure you would like to give up?", timeout=20)
+        dbcog = await self.get_dbcog()
+        if not confirmation:
+            if confirmation is None:
+                await ctx.send("Confirmation timeout.", delete_after=10)
+            return
+        monster = await dbcog.find_monster(str(await self.config.padleToday()), ctx.author.id)
+        await self.config.user(ctx.author).done.set(True)
+        monsterEmbed = EmbedView(EmbedMain(title=MonsterHeader.menu_title(monster).to_markdown(),
+                                           description="PADle #{}".format(await self.config.numDays()),
+                                           url=MonsterLink.ilmina(monster)),
+                                 embed_thumbnail=EmbedThumbnail(MonsterImage.icon(monster.monster_id))).to_embed()
+        await ctx.send(embed=monsterEmbed)
+        await ctx.send("Use `^padle score` to share your score!")
+        async with self.config.user(ctx.author).score() as scores:
+            scores.append(":x:")
+        async with self.config.allScores() as allScores:
+            allScores.append("X")
+
+    @padle.command()
+    async def score(self, ctx):
+        """Share your PADle score"""
+        if(await self.config.user(ctx.author).done()):
```

Comment:
change this to if not done then send cancellation message you have not done, return

and then you can unindent the entire rest of the function a block